Services/DataBaseService/Utils/MigrationScripts/24.03.2020/0001_CreateUsersTable.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE [UsersCredentials] (
+    [Id]		    UNIQUEIDENTIFIER NOT NULL   UNIQUE,
+    [Email]		    NVARCHAR (30)    NOT NULL   UNIQUE CLUSTERED,
```

Comment:
Why do we have restriction on email length so small?

Reply:
raised to 50